standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -638,12 +638,12 @@ public enum ConfVars {
         "hive.txn.acid.metrics.delta.pct.threshold", 0.01f,
         "Percentage (fractional) size of the delta files relative to the base directory. Deltas smaller than this threshold " +
             "count as small deltas. Default 0.01 = 1%.)"),
-    COMPACTOR_INITIATOR_ON("metastore.compactor.initiator.on", "hive.compactor.initiator.on", false,
+    COMPACTOR_INITIATOR_ON("metastore.compactor.initiator.on", "hive.compactor.initiator.on", true,
         "Whether to run the initiator thread on this metastore instance or not.\n" +
             "Set this to true on one instance of the Thrift metastore service as part of turning\n" +
             "on Hive transactions. For a complete list of parameters required for turning on\n" +
             "transactions, see hive.txn.manager."),
-    COMPACTOR_CLEANER_ON("metastore.compactor.cleaner.on", "hive.compactor.cleaner.on", false,
+    COMPACTOR_CLEANER_ON("metastore.compactor.cleaner.on", "hive.compactor.cleaner.on", true,
```

Comment:
If you set it to `true` all the HMS instances will have it turned on, maybe that ain't the intention of this PR, IIRC we usually enable cleaner only on one HMS instance not all, which post this change will be enabled on all

Reply:
We would have only one instance running the `initiator` and `cleaner`, as well as the housekeeping tasks, as the `metastore.housekeeping.leader.election` is changed to `lock`, all HMS instances will trying to acquire the exclusive Hive lock, only the winner will become the leader and run these maintenance tasks.

Reply:
ACID is the default table type so that makes sense. However, once we switch to Iceberg, I don't think we should keep ACID housekeeping services enabled by default.

Reply:
@dengzhhu653 keep in mind that once that's in, you'll need to explicitly disable these configs for some of the form factors downstream.